tmt/export.py

```diff
@@ -46,21 +46,22 @@ def export_to_nitrate(test, create, general):
     try:
         nitrate_id = test.node.get('extra-nitrate')[3:]
         nitrate_case = nitrate.TestCase(int(nitrate_id))
-        nitrate_case.summary # Make sure we connect to the server now
+        nitrate_case.summary  # Make sure we connect to the server now
         echo(style(f"Test case '{nitrate_case.identifier}' found.", fg='blue'))
     except TypeError:
         # Create a new nitrate test case
         if create:
             nitrate_case = create_nitrate_case(test)
             new_test_created = True
+            test._metadata['extra-summary'] = nitrate_case.summary
         else:
             raise ConvertError("Nitrate test case id not found.")
     except (nitrate.NitrateError, gssapi.raw.misc.GSSError) as error:
         raise ConvertError(error)
 
     # Summary
-    summary = test.node.get(
-        'extra-summary', test.node.get('extra-task', test.summary))
+    summary = test._metadata.get('extra-summary') \
```

Comment:
Not very PEP8 friendly - https://www.python.org/dev/peps/pep-0008/#should-a-line-break-before-or-after-a-binary-operator

Reply:
Maybe I am reading the PEP wrong but doesn't it say that break should be before the operator (i.e. the way it's done in this PR)?
Anyway this was created by `autopep` so I assumed it was correct. I can change it if you insist.

Reply:
Yeah, I would suggest to use `()` to group long expressions across multiple lines.

Reply:
Fixed in 5db6f23